aplib.net-demo/Assets/Scripts/Player/Movement.cs

```diff
@@ -0,0 +1,191 @@
+using UnityEngine;
+
+public class Movement : MonoBehaviour
+{
+    [SerializeField] private float _maxSpeed = 7;
+    [SerializeField] private float _acceleration = 100;
+    private Vector3 _horizontalVelocity;
+    private Vector2 _horizontalInput;
+
+    [SerializeField] private float _normalGravityScale = 1.7f;
+    [SerializeField] private float _fallingGravityScale = 2.5f;
+    private float _actualGravityScale;
+    private float _gravity;
+
+    [SerializeField] private float _jumpHeight = 7;
+    [SerializeField] private float _airMovementScale = 0.5f;
+    private bool _jumpPressed;
+
+    [SerializeField] private LayerMask _groundMask;
+    [SerializeField] private float _groundDrag = 0.7f;
+    [SerializeField] private float _slopeAngle = 40;
+    private float _playerHeight;
+    private float _playerRadius;
+    private Vector3 _bottomPoint;
+    private bool _isGrounded;
+
+    private Color _velocityColor = Color.red;
+    private Color _physicsColor = Color.blue;
```

Comment:
We probably want to define these globally, to allow for consistent colours across the entire project.